flink-python/pyflink/datastream/window.py

```diff
@@ -368,7 +368,7 @@ def get_metric_group(self) -> MetricGroup:
             """
             Returns the metric group for this :class:`Trigger`. This is the same metric group that
             would be returned from
-            :func:`~pyflink.datasteam.functions.RuntimeContext.get_metric_group` in a user function.
+            :func:`~pyflink.datastream.functions.RuntimeContext.get_metric_group` in a user function.
```

Comment:
![image](https://github.com/user-attachments/assets/5e69f2da-3b95-4fa3-a738-ce067229696c)


This line's comment is too long, exceeding the threshold and causing CI failure.